Doc/#1209 - #1250 - Add documentation page for each application and page template (#1263)
Avaiga/taipy-doc: docs/userman/ecosystem/cli/create.md

# Create a Taipy application from a template

Taipy provides a comfortable environment for getting started with Taipy via the create command,
and is the best way to start building a new application with Taipy.

To create a simple Taipy application, you can run `taipy create` from the CLI, then answer a few
questions to customize your application.
```console
$ taipy create
```

> By default, the `taipy create` command helps a new Taipy application using the default Taipy application template.

Reviewer: ```suggestion
By default, the `taipy create` command helps scaffolding a new Taipy application using the
default Taipy application template.
```

You can specify creating a new application from another application template using the
*--application* option.

```console
$ taipy create --application sdm
```

Taipy currently supports the following application templates:

- [Default application template](../templates/applications/default_app.md)
- [Scenario management application template](../templates/applications/sdm_app.md)

> # List of application templates

Reviewer: ```suggestion
# List the application templates
```

> From the CLI, you can list all supported application templates by running `taipy help create`

Reviewer: ```suggestion
From the CLI, you can list the available application templates by running `taipy help create`
```

# Create a Taipy application from a template

Taipy provides a comfortable environment for getting started with Taipy via the create command,
and is the best way to start building a new application with Taipy.

To create a simple Taipy application, you can run `taipy create` from the CLI, then answer a few
questions to customize your application.
```console
$ taipy create
```
By default, the `taipy create` command helps scaffolding a new Taipy application using the
default Taipy application template.

You can specify creating a new application from another application template using the
*--application* option.

```console
$ taipy create --application sdm
```

Taipy currently supports the following application templates:

- [Default application template](../tp_templates/applications/default_app.md)
- [Scenario management application template](../tp_templates/applications/sdm_app.md)

> # List the application templates

Reviewer 2: ```suggestion
# Application templates list
```
